contracts/registry/ReverseRegistrar.sol

```diff
@@ -102,9 +119,15 @@ contract ReverseRegistrar {
         addr;
```

Comment:
We can probably remove these lines; I think Solidity should correctly detect that they're used now.